### PR TITLE
Implement minimal GET app delete job

### PR DESCRIPTION
**Pull request by acosta11**
## Is there a related GitHub Issue?
[#335](https://github.com/cloudfoundry/cf-k8s-controllers/issues/335)

## What is this change about?
Add a minimal implementation of the GET jobs endpoint for app delete jobs.

## Does this PR introduce a breaking change?
No

## Acceptance Steps
See related issue.

## Tag your pair, your PM, and/or team
paired with @Birdrock

### CODE REVIEW

File: api/apis/job_handler.go [hunk 1]
```diff
@@ -34,26 +36,27 @@ func (h *JobHandler) jobGetHandler(w http.ResponseWriter, r *http.Request) {
...
 	}
-}
 
-func getSpaceGUID(jobGUID string) string {
-	if strings.HasPrefix(jobGUID, syncSpacePrefix) {
-		spaceGUID := strings.Replace(jobGUID, syncSpacePrefix, "", 1)
-		return spaceGUID
-	}
-	return ""
+	w.WriteHeader(http.StatusOK)
```
Comment (davewalter) [inline api/apis/job_handler.go:58]: I think we normally omit this as this is the default status code.

**Review by davewalter — APPROVED** [review]
Looks good. Just the comment about the explicit 200 OK response code.

File: api/apis/job_handler.go (hunk 1 above)
Comment (acosta11) [reply api/apis/job_handler.go:58]: Good call I think I missed this when rebasing on main along with some other artifacts of letting the branch languish over the weekend. Will remove.

**Comment by matt-royal** [comment]
Since it seemed like the StatusOK line was the only issue, @julian-hj and I went ahead and removed it and force pushed over the existing branch. We'll merge it once all the tests pass

**Pull request merged**